python/examples/webhook-server/main.py

```diff
@@ -0,0 +1,70 @@
+# std imports
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from http import HTTPStatus
+from typing import Mapping
+
+# third-party imports
+import requests
+from truelayer_signing import HttpMethod, extract_jws_header, verify_with_jkws
+
+HOOK_PATH: str = "/hook/d7a2c49d-110a-4ed2-a07d-8fdb3ea6424b"
+
+
+class HookHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        if self.path == HOOK_PATH:
+            content_length = int(self.headers['Content-Length'])
+            body = self.rfile.read(content_length)
+
+            try:
+                res = hook_handler(self.path, self.headers, body.decode())
+                self.send_response(res)
+            except Exception as err:
```

Comment:
I'd make TL verification functions return a specific exception type (or exceptions deriving from a common base) so that we could return `UNAUTHORIZED` only for actual signature validation failures, not errors in `send_response` or or other errors